src/cljc/athens/common_events.cljc

```diff
@@ -177,14 +177,20 @@
   - `parent-uid`: `:block/uid` of parent block
   - `new-uid`  : new child's block uid
   - `add-time?`: Should `:edit/time` for this block be transacted"
-  [last-tx parent-uid new-uid add-time?]
-  (let [event-id (gen-event-id)]
-    {:event/id      event-id
-     :event/last-tx last-tx
-     :event/type    :datascript/add-child
-     :event/args    {:parent-uid parent-uid
-                     :new-uid    new-uid
-                     :add-time?  add-time?}}))
+  ([last-tx parent-uid new-uid] (let [event-id (gen-event-id)]
+                                  {:event/id      event-id
+                                   :event/last-tx last-tx
+                                   :event/type    :datascript/add-child
+                                   :event/args    {:parent-uid parent-uid
+                                                   :new-uid    new-uid
+                                                   :add-time?  false?}}))
```

Comment:
Should this be `false` instead of `false?`

Reply:
Thank you Junius, yeah it should be

Reply:
Good catch, very subtle bug!